woocommerce/woocommerce-gateway-stripe
# feat: add payment method icons

## Changes proposed in this Pull Request:

Adding some base payment method icons for reuse.

## Testing instructions

- Ensure you have the `_wcstripe_feature_upe_settings` feature flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- You should see a list of icons with their 2 different sizes
![Screen Shot 2021-08-20 at 12 34 12 PM](https://user-images.githubusercontent.com/273592/130271962-54bcb196-39f1-46d3-b0df-dd48150c432c.png)
